plugins/available/base.plugin.bash

```diff
@@ -4,13 +4,24 @@ about-plugin 'miscellaneous tools'
 function ips ()
 {
     about 'display all ip addresses for this host'
+    param 'optional -i'
```

Comment:
Could I write "param 'optional -i , display the interfaces and the corresponding ip addresses '  "? or other way to do it?